pkg/client/dynamic/clientset/tekton/tekton.go

```diff
@@ -25,5 +32,15 @@ func WithClient(client dynamic.Interface) clientset.Option {
 				cs.Add(r, client)
 			}
 		}
+		for version, resources := range allowedTriggersTypes {
```

Comment:
Do we need TriggersType here? @dibyom

Reply:
This is needed to pass now, since everything is going over the dynamic client now.

Reply:
If everything is going over the dynamic client now, can we get rid of the typed clientsets in the Sink struct?